x-pack/plugins/upgrade_assistant/public/application/components/es_deprecations/deprecation_types/reindex/flyout/progress.test.tsx

```diff
@@ -33,7 +33,7 @@ describe('ReindexProgress', () => {
   steps={
     Array [
       Object {
-        "status": "incomplete",
```

Comment:
Why did this need to change?

Reply:
In the function that renders the [stepDetails](https://github.com/elastic/kibana/blob/c25a569c66c3bd0d7c536e25ba5c1c8bcde87f6c/x-pack/plugins/upgrade_assistant/public/application/components/es_deprecations/deprecation_types/reindex/flyout/progress.tsx#L122) we compute the `lastStep` by finding the index of the current step in the `ReindexStep` enum and subtracting 1 from it. 

Since the `ReindexStep` enum use to be in the following form:
```
...
  created = 0,
  indexGroupServicesStopped = 10,
  readonly = 20,
...
```
For the first step (readonly), the `previousStep` was then computed as `indexGroupServicesStopped` and since the test states that the reindexing is `inProgress` and that the last step is `ReindexStep.created`, this was always marked as `incomplete` as per: https://github.com/elastic/kibana/blob/c25a569c66c3bd0d7c536e25ba5c1c8bcde87f6c/x-pack/plugins/upgrade_assistant/public/application/components/es_deprecations/deprecation_types/reindex/flyout/progress.tsx#L139-L142

We have now removed `indexGroupServicesStopped`, so the above logic now results in `currentStep=created` and `previousStep=created` which ends up resolving as `inProgress`:

https://github.com/elastic/kibana/blob/c25a569c66c3bd0d7c536e25ba5c1c8bcde87f6c/x-pack/plugins/upgrade_assistant/public/application/components/es_deprecations/deprecation_types/reindex/flyout/progress.tsx#L143-L146

I thought the change was good for UX since its expected that after starting the reindexing process the first step should be marked in the UI as in progress as soon as possible. WDYT?

Reply:
It's a bit convoluted but I think I understand. :) Thanks for explaining, sounds good.